### ISSUE TITLE
Starting point for a "delete" button extension?

### ISSUE THREAD

**Issue by kurtextrem** [FreshRSS/FreshRSS#3149]
Hey all, thanks for developing this. Youtube recently deprecated notification emails, so many people are looking for a replacement for that now.
So FreshRSS basically has everything like an email client, but one big thing is missing: Deleting single entries. Which is why I want to develop an extension for that. 

Where should I look for the following things:
a) How to add a new button next to the star or read/unread icon?
b) Is there a function yet for deleting entries?

**Comment by Alkarex** [comment]
Hello,
Would maybe automatically marking some articles as read be sufficient? Supplemented by the existing regular house-keeping that auto-deletes old articles?

**Comment by kurtextrem** [comment]
Yeah, maybe. I thought about it too: Star what shouldn't be deleted and then unread what should be deleted. But this is more of a two-step workaround instead of a single click to delete and just keep the rest like it is, I'd say.

**Comment by Frenzie** [comment]
There's discussion about this kind of thing somewhere btw, but this is just a quick drive-by, not going to look for it atm. ;-)

**Comment by Alkarex** [comment]
Ah, you mean actions from the user (and not automatic)? You can control what will be deleted or not by the automatic house-keeping in the global or feed settings. So isn't simply marking as read sufficient?

**Comment by Alkarex** [comment]
Marking as read is safer, because deleting a recent article comes with the risk of receiving it again if it still exists in the original RSS feed

**Comment by kurtextrem** [comment]
Well yeah, but usually you want to have some kind of archive. If you can't remove unwanted articles, it's not so much of an useful archive anymore. Maybe a "hide from archive" would be useful then - or a mass selection feature, where you can move read stuff into the label "Trash", which you can clear afterwards?

**Comment by wrobelda** [comment]
>  Maybe a "hide from archive" would be useful then - or a mass selection feature, where you can move read stuff into the label "Trash", which you can clear afterwards?

So I guess what you're suggesting here is basically a Trash, which one could configure to automatically empty after a while – but only that, keeping the rest of archive in place. And then letting user move any article to Trash would do.

**Comment by math-GH** [comment]
How is the current state here?
